5calls/app/src/main/res/layout/activity_about.xml

```diff
@@ -1,177 +1,92 @@
-            android:layout_margin="0dp"
-            android:background="@android:color/transparent"
-            android:minHeight="@dimen/accessibility_min_size"
-            android:text="@string/sign_up_newsletter_btn"
-            android:textAlignment="textStart"
-            android:textColor="@color/colorAccent"
-            />
+            style="@style/transparentBtnStyle"
+            android:text="@string/sign_up_newsletter_btn" />
 
         <TextView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            style="@style/aboutParagraphTextStyle"
             android:text="@string/about_us_prompt"
-            android:textColor="@color/colorPrimary"
-            android:textSize="@dimen/text_size_paragraph"
-            android:textStyle="bold|italic"
             />
 
         <Button
             android:id="@+id/about_us_btn"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="start"
-            android:layout_margin="0dp"
-            android:background="@android:color/transparent"
-            android:minHeight="@dimen/accessibility_min_size"
+            style="@style/transparentBtnStyle"
             android:text="@string/about_us_btn"
-            android:textAlignment="textStart"
-            android:textColor="@color/colorAccent"
             />
 
         <TextView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            style="@style/aboutParagraphTextStyle"
             android:text="@string/contact_us_prompt"
-            android:textColor="@color/colorPrimary"
-            android:textSize="@dimen/text_size_paragraph"
-            android:textStyle="bold|italic"
-        />
+            />
 
         <Button
             android:id="@+id/contact_us_btn"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="start"
-            android:layout_margin="0dp"
-            android:background="@android:color/transparent"
-            android:minHeight="@dimen/accessibility_min_size"
+            style="@style/transparentBtnStyle"
             android:text="@string/contact_us_btn"
-            android:textAlignment="textStart"
-            android:textColor="@color/colorAccent"
-        />
-
-        <TextView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:text="@string/rate_us_prompt"
-            android:textColor="@color/colorPrimary"
-            android:textSize="@dimen/text_size_paragraph"
-            android:textStyle="bold|italic"
             />
 
+        <TextView
+            style="@style/aboutParagraphTextStyle"
+            android:text="@string/rate_us_prompt" />
+        <!-- wasn't in next btn <item name="android:layout_margin">@dimen/layout_margin_button</item>-->
```

Comment:
Can you resolve or remove this comment?

Reply:
I forgot to remove it, but now it is gone.